packages/@okta/vuepress-site/docs/api/resources/zones/index.md

```diff
@@ -2,23 +2,160 @@
 title: Zones
 category: management
 ---
-
 # Zones API
 
 > This API is an <ApiLifecycle access="ea" /> feature.
 
 The Okta Zones API provides operations to manage zones in your organization. Zones may be used to guide policy decisions.
 
-IP zones are the only type currently supported by the Zones API. IP zones are used to group IP address ranges so that policy decisions can be made based on the client's IP location.
+## Zone Model
+
+### Base Network Zone Properties
+
+The Base Network Zone object defines the base properties which concrete Network Zone types (IP and Dynamic) inherit.
```

Comment:
I think the wording could be changed to not sound too object hierarchy oriented. How about saying that the "These are the common properties for all network zones.." or something like that?

@jakubvul can you please review the wording here and let us know if there are existing patterns within our docs that we can reuse to describe this hierarchy.

Reply:
I think "the common properties for all" or "the properties shared by all" both work.